src/odemis/dataio/tiff.py

```diff
@@ -3133,3 +3142,101 @@ def _convert_tescan_to_odemis_metadata(metadata: bytes) -> dict:
     
     return md
 
+def _convert_openfibsem_to_odemis_metadata(metadata: str) -> dict:
+
+    # load json metadata
+    omd = json.loads(metadata)        
+    md = {}
+
+    # general
+    try:
+        timestamp = omd["microscope_state"]["timestamp"]
+        md[model.MD_ACQ_DATE] = timestamp
+    except Exception as e:
+        logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_ACQ_DATE} - {type(e)}: {e}")
+
+    try:
+        system_info = omd["system"]["info"]
+        md[model.MD_HW_NAME] = system_info["manufacturer"] + " " + system_info["model"] + "-" + system_info["serial_number"]  
+    except KeyError as e:
+        logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_HW_NAME} -  {type(e)}: {e}")
+    try:
+        system_info = omd["system"]["info"]
+        md[model.MD_HW_VERSION] = system_info["hardware_version"] 
+    except KeyError as e:
+        logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_HW_NAME} -  {type(e)}: {e}")
+    
+    try:
+        md[model.MD_SW_VERSION] = omd["experiment"]["fibsem_version"]
+    except:
+        logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_SW_VERSION} - {type(e)}: {e}")
+    
+    try:
+        md[model.MD_PIXEL_SIZE] = (float(omd["pixel_size"]["x"]), 
+                                   float(omd["pixel_size"]["y"]))
+    except KeyError as e:
+        logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_PIXEL_SIZE} - {type(e)}: {e}") 
+
+    # beam metadata
+    try:
+        beam_type = omd["image"]["beam_type"]
+        try:
+            md[model.MD_DWELL_TIME] = float(omd["image"]["dwell_time"])
+        except KeyError as e:
+            logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_DWELL_TIME} - {e}")
+        try:
+            # MD_POS should match image dimensions, e.g. only XY for 2D, XYZ for 3D, etc.
+            md[model.MD_POS] = (float(omd["microscope_state"]["stage_position"]["x"]),
+                                float(omd["microscope_state"]["stage_position"]["y"])
+                                )
+        except KeyError as e:
+            logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_POS} - {e}")
+
+        try:   
+            acq_map =  {
+                "ELECTRON": model.MD_AT_EM,
+                "ION": model.MD_AT_FIB,
+            }
+            md[model.MD_ACQ_TYPE] = acq_map[beam_type]
+
+        except KeyError as e:
+            logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_ACQ_TYPE} - {e}")
+
+        beam_type_settings = "electron_beam" if beam_type == "ELECTRON" else "ion_beam"
+        beam_settings = omd["microscope_state"][beam_type_settings]
+            
+        try:
+            md[model.MD_ROTATION] = numpy.deg2rad(float(beam_settings["scan_rotation"]))
```

Comment:
math.radians should be fine here
```suggestion
            md[model.MD_ROTATION] = math.radians(float(beam_settings["scan_rotation"]))
```